cmd/metal-api/internal/service/partition-service_test.go

```diff
@@ -182,7 +187,7 @@ func TestUpdatePartition(t *testing.T) {
 	container := restful.NewContainer().Add(service)
 
 	mgmtService := "mgmt"
-	imageURL := "http://somewhere/image1.zip"
+	downloadableFile := "https://upload.wikimedia.org/wikipedia/commons/e/e6/1kb.png"
```

Comment:
Somehow I don't like so much relying on the internet for unit tests. Sometimes I develop code on a travel or something and then your tests start failing when you're in an airplane. 😅
Don't you think it would be better to direct this to a local endpoint?

Reply:
Right, fixed.